src/igvfd/loadxl.py

```diff
@@ -525,7 +525,10 @@ def get_pipeline(testapp, docsdir, test_only, item_type, phase=None, method=None
         remove_keys('lab', 'submits_for'),
     ],
     'construct_library_set': [
-        remove_keys('integrated_content_files'),
+        remove_keys('integrated_content_files', 'large_scale_gene_list', 'large_scale_loci_list'),
+    ],
+    'prediction_set': [
+        remove_keys('large_scale_gene_list', 'large_scale_loci_list', 'scope'),
```

Comment:
scope is not linked object, why do you ned to remove it?

Reply:
It is necessary to get the insert to load properly. https://github.com/IGVF-DACC/igvfd/pull/731#discussion_r1458020172